modules/redis/scanner.go

```diff
@@ -212,12 +335,23 @@ func (s *Scanner) Protocol() string {
 	return "redis"
 }
 
+// Converts the string to a Uint32 if possible. If not, returns 0 (the zero value of a uin32)
+func convToUint32(s string) uint32 {
+	s_64, err := strconv.ParseUint(s, 10, 32)
+	if err != nil {
+		// TODO: LEARN HOW TO LOG
```

Comment:
Usually bad data errors like this wouldn't get logged -- either it's not bad enough to make you fail detection, but instead just leave something out of the result (so the conversion error can be inferred from the absence of that field), or it is bad enough to fail, so you return e.g. a protocol error and put your log message in the error string.